src/coreComponents/physicsSolvers/fluidFlow/LogLevelsInfo.hpp

```diff
@@ -48,6 +48,24 @@ struct Statistics
   static constexpr std::string_view getDescription() { return "Print statistics"; }
 };
 
+struct sourceFluxRegionsStatistics
+{
+  static constexpr int getMinLogLevel() { return 3; }
+  static constexpr std::string_view getDescription() { return "Print source flux statistics for each regions "; }
+};
+
+struct sourceFluxStatistics
+{
+  static constexpr int getMinLogLevel() { return 2; }
+  static constexpr std::string_view getDescription() { return "Print source flux statistics for each flux in a mesh"; }
+};
+
+struct sourceFluxMeshStatistics
+{
+  static constexpr int getMinLogLevel() { return 1; }
+  static constexpr std::string_view getDescription() { return "Print source flux statistics resulting from all flux in a mesh"; }
```

Comment:
these two are confusing - not clear description

Reply:
Here is my suggestion:
```
struct allSourceFluxesStats
{
  static constexpr int getMinLogLevel() { return 1; }
  static constexpr std::string_view getDescription() { return "Print aggregated statistics over all source fluxes on the whole mesh"; }
};

struct sourceFluxStats
{
  static constexpr int getMinLogLevel() { return 2; }
  static constexpr std::string_view getDescription() { return "Print statistics for each source flux over the whole mesh"; }
};

struct regionSourceFluxStats
{
  static constexpr int getMinLogLevel() { return 3; }
  static constexpr std::string_view getDescription() { return "Print statistics for each source flux in each region"; }
};
```

Reply:
yeah, these descriptions sounds good
please also try to structs names to better match the descriptions